README.md

```diff
@@ -1 +1,187 @@
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [FIFA] <a name="about-project"></a>
+
+> This show the FIFA & and QATAR WORLD CUP 2022
+>and also other champions realease in that date.
+
+**[FIFA]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+-JavaScript
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Functionality_to_navbar]**
+- **[Dynamic_page_Featured_Game]**
+- **[Separate_create_Home_About]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://ilyassatlassi.github.io/capstone/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/ilyassatlassi/capstone.git
+```
+-
+
+### Install
+
+Install this project with:
+This is not recommad but you can check the linterc error in your local pc
+
+<!--
+Example command:
+--->
+```sh
+  cd my-project
+  npm install --save-dev hint@7.x
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 ilyass atlassi
+
+- GitHub: [@githubhandle](https://github.com/ilyassatlass)
+- Twitter: [@twitterhandle](https://twitter.com/ilyass_atlassi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ilyassatlassi/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ OPTIONAL] It would be nice to add `Future Features` according to the design of the template. The Future Features section should have 1-3 features that can be added to this project. This way, developers who want to collaborate with you on this project will find it easy to do so because they know what features to add. 👍